gobusterdir/options.go

```diff
@@ -27,6 +27,7 @@ type OptionsDir struct {
 	UseSlash          bool
 	IsWildcard        bool
 	WildcardForced    bool
+	HeadRequests      bool
```

Comment:
So before we go ahead with this, should we discuss the handling of a wider range of verbs?